sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -4,12 +4,25 @@
 
 ### Features Added
 
+* Added `Resource` to traces, logs, and metrics with default configuration.
+  ([#37837](https://github.com/Azure/azure-sdk-for-net/pull/37837))
+* Added package `OpenTelemetry.ResourceDetectors.Azure` to introduce OpenTelemetry Resource detectors `AppServiceResourceDetector` and `AzureVMResourceDetector`.
```

Comment:
```suggestion
* Added resource detection for `Azure App Service` and `Azure VM` environment. 
```

Reply:
Do we need a readme update on how to override these?

Reply:
Yes, we should a follow up PR.

Reply:
@TimothyMothra Could you please help here once this PR gets merged?

Reply:
This is covered already in readme - https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore#adding-custom-resource

I tested and it works.

Reply:
Should we document `builder.Services.AddOpenTelemetry().ConfigureResource(resourceBuilder => resourceBuilder.AddService("service-name"));` which is common for all 3 signals.